package.json

```diff
@@ -45,14 +46,16 @@
     "eslint-plugin-mocha": "^10.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eth-block-tracker": "^8.1.0",
     "ethereumjs-util": "^6.1.0",
     "mocha": "^9.2.2",
     "prettier": "^2.7.1",
     "prettier-plugin-packagejson": "^2.2.11",
     "typescript": "^4.0.7",
     "xtend": "^4.0.1"
   },
+  "peerDependencies": {
+    "@metamask/eth-block-tracker": ">=9"
```

Comment:
Defaulting to a more liberal range here so by default new major releases won't require bumping this.